index.html

```diff
@@ -58,5 +58,179 @@ <h1>I'm Collins. Glad to see you!</h1>
+              <li class="listB-items">Css</li>
+              <li class="listB-items">JavaScript</li>
+            </ul>
+            <button>See Project</button>
+          </div>
+        </div>
+      </div>
+      <div class="grid-items">
+        <div class="cards">
+          <img src="./Images/Snapshoot-Portfolio (2).png" alt="" />
+        </div>
+        <div class="cards">
+          <h1>Tonic</h1>
+          <div class="cards-details">
+            <div class="project">
+              <span>Canopy</span>
+              <ul class="listA">
+                <li class="listA-items">BackEnd</li>
+                <li class="listA-items">2015</li>
+              </ul>
+            </div>
+
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="listB">
+              <li class="listB-items">HTML</li>
+              <li class="listB-items">Css</li>
+              <li class="listB-items">JavaScript</li>
+            </ul>
+            <button>See Project</button>
+          </div>
+        </div>
+      </div>
+      <div class="grid-items">
+        <div class="cards">
+          <img src="./Images/Snapshoot-Portfolio (3).png" alt="" />
+        </div>
+        <div class="cards">
+          <h1>Multi-Post Stories</h1>
+          <div class="cards-details">
+            <div class="project">
+              <span>Canopy</span>
+              <ul class="listA">
+                <li class="listA-items">BackEnd</li>
+                <li class="listA-items">2015</li>
+              </ul>
+            </div>
+
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="listB">
+              <li class="listB-items">HTML</li>
+              <li class="listB-items">Css</li>
+              <li class="listB-items">JavaScript</li>
+            </ul>
+            <button>See Project</button>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="about-myself">
+      <div class="details">
+        <h1 class="sectionheading">About Myself</h1>
+
+        <div class="about-myself-paragraph">
+          <p class="myself">
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hesitate to
+            contact me.
+          </p>
+        </div>
+        <span class="socialmedia">LETS CONNECT</span>
+        <ul class="media">
+          <li><i class="fa-brands fa-twitter"></i></li>
+          <li><i class="fa-brands fa-linkedin-in"></i></li>
+          <li><i class="fa-brands fa-github"></i></li>
+          <li><i class="fa-brands fa-facebook-f"></i></li>
+        </ul>
+        <button class="resume-button">Get my resume</button>
```

Comment:
the about myself section looks good 👍🏼  , some small things to fixe to make it perfect, the vertical spacing between these two buttons and the social links list is not equal, the text in the paragraph and in Lets connect  is a bit bolder than the design and the spacing between the social icons should be increased.

![image (36)](https://github.com/enocol/MobileFirst/assets/59707859/ec3300b5-2e1e-4b1f-b19d-a39148876f39)
![image (37)](https://github.com/enocol/MobileFirst/assets/59707859/8503f1f7-52d7-4e69-b499-70c76c43db5f)